src/plots/smith/layout_attributes.js

```diff
@@ -0,0 +1,235 @@
+    linewidth: axesAttrs.linewidth,
+    showgrid: extendFlat({}, axesAttrs.showgrid, {dflt: true}),
+    gridcolor: axesAttrs.gridcolor,
+    gridwidth: axesAttrs.gridwidth
+}, 'plot', 'from-root');
+
+var axisTickAttrs = overrideAll({
+    tickmode: axesAttrs.tickmode,
+    tickvals: axesAttrs.tickvals,
+    ticktext: axesAttrs.ticktext,
+    ticks: axesAttrs.ticks,
+    ticklen: axesAttrs.ticklen,
+    tickwidth: axesAttrs.tickwidth,
+    tickcolor: axesAttrs.tickcolor,
+    showticklabels: axesAttrs.showticklabels,
+    showtickprefix: axesAttrs.showtickprefix,
+    tickprefix: axesAttrs.tickprefix,
+    showticksuffix: axesAttrs.showticksuffix,
+    ticksuffix: axesAttrs.ticksuffix,
+    showexponent: axesAttrs.showexponent,
+    exponentformat: axesAttrs.exponentformat,
+    minexponent: axesAttrs.minexponent,
+    separatethousands: axesAttrs.separatethousands,
+    tickfont: axesAttrs.tickfont,
+    tickangle: axesAttrs.tickangle,
+    tickformat: axesAttrs.tickformat,
+    tickformatstops: axesAttrs.tickformatstops,
+    layer: axesAttrs.layer
+}, 'plot', 'from-root');
+
+var radialAxisAttrs = {
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    angle: {
+        valType: 'angle',
+        editType: 'plot',
+        description: [
+            'Sets the angle (in degrees) from which the radial axis is drawn.',
+            'Note that by default, radial axis line on the theta=0 line',
+            'corresponds to a line pointing right (like what mathematicians prefer).',
+            'Defaults to the first `polar.sector` angle.'
+        ].join(' ')
+    },
+
+    side: {
+        valType: 'enumerated',
+        // TODO add 'center' for `showline: false` radial axes
+        values: ['clockwise', 'counterclockwise'],
+        dflt: 'clockwise',
+        editType: 'plot',
+        description: [
+            'Determines on which side of radial axis line',
+            'the tick and tick labels appear.'
+        ].join(' ')
+    },
+
+
+    title: {
+        // radial title is not gui-editable at the moment,
+        // so it needs dflt: '', similar to carpet axes.
+        text: extendFlat({}, axesAttrs.title.text, {editType: 'plot', dflt: ''}),
+        font: extendFlat({}, axesAttrs.title.font, {editType: 'plot'}),
+
+        // TODO
+        // - might need a 'titleside' and even 'titledirection' down the road
+        // - what about standoff ??
+
+        editType: 'plot'
+    },
+
+    hoverformat: axesAttrs.hoverformat,
+
+    uirevision: {
+        valType: 'any',
+        editType: 'none',
+        description: [
+            'Controls persistence of user-driven changes in axis `range`,',
+            '`autorange`, `angle`, and `title` if in `editable: true` configuration.',
+            'Defaults to `polar<N>.uirevision`.'
+        ].join(' ')
+    },
+
+    editType: 'calc',
+
+    _deprecated: {
+        title: axesAttrs._deprecated.title,
+        titlefont: axesAttrs._deprecated.titlefont
```

Comment:
Please do not implement any `_deprecated` attributes for these new features.

Reply:
See 090251f067dfb25130e0afa956e8d87d19eb3d00.